docs/source/index.rst

```diff
@@ -47,6 +47,7 @@ Contributors:
 * Victor Safronovich <vsafronovich@gmail.com>
 * Dominik Kozaczko <http://dominik.kozaczko.info>
 * Petr Dlouhý <petr.dlouhy@email.cz>
+* Blender Kit <info@blenderkit.com>
```

Comment:
`BlenderKit <admin@blenderkit.com>` and/or give yourself a credit

Reply:
Changed